### PR TITLE
Pathway editor controls

### PR DESCRIPTION
**Pull request by stenington**
This _just_ styles the editor controls, and widens the editor in desktop and tablet views.

### CODE REVIEW

**Comment by stenington** [comment]
http://discovery-286.herokuapp.com/

**Comment by stenington** [comment]
Obviously one issue is
- [x] missing undo button svg

File: clientapp/styles/pathway-editor.scss [hunk 1]
```diff
@@ -1,7 +1,87 @@
...
+    }
+  }
+
+  .editor-add {
+    @include icon("/static/pathway/icon-add-badge.svg", "/static/pathway/icon-add-badge-active.svg");
+  }
+  .editor-delete {
+    @include icon("/static/pathway/icon-remove-parts.svg", "/static/pathway/icon-remove-parts-active.svg");
+  }
 }
```
Comment (ZeeJab) [inline clientapp/styles/pathway-editor.scss:89]: this is awesome!

File: clientapp/templates/includes/editor.html [hunk 2]
```diff
@@ -1,8 +1,19 @@
 <div>
   {% if showControls %}
-    <a href="#" class="button" data-stack="addPanel" >Add</a>
-    <a href="#" class="button button-alert js-delete" data-toggle="off">Delete</a>
-    <a href="#" class="button button-secondary js-undo">Undo</a>
+  <ul class="editor-controls">
+    <li class="editor-button editor-add js-editor-add" data-stack="addPanel">
+      <button/>
```
Comment (ZeeJab) [inline clientapp/templates/includes/editor.html]: adding a type to this makes it more browser freindly: doing `<button type="button">` works better with firefox 29 and bellow more specifically. Also why aren't you doing it as `<button>Add</button>`?

**Comment by ZeeJab** [comment]
besides the `button` thing which i will leave up to you to decide what to do with, I think we should merge this in... ?

File: clientapp/templates/includes/editor.html (hunk 2 above)
Comment (stenington) [inline clientapp/templates/includes/editor.html]: Yeah, the empty button tag is weird, I agree. Didn't know about the `type="button"` thing so I'm happy to add that in.

The empty button thing had to do with being able to set the z-index for the label to a lower layer than the button so the label could slide under the button, basically. I think I can achieve the same effect with slightly more canonical HTML though, so I'll spend a few minutes on that.

**Comment by stenington** [comment]
@ZeeJab Okay, no more empty button tags! The button name is still duplicated in the label, but I'm not too bothered by that. What do you think?

@zzzndr Do you have button artwork for "undo"?

**Comment by ZeeJab** [comment]
@stenington Perf!

**Comment by zzzndr** [comment]
we did!
here are the links to dropbox:
https://www.dropbox.com/s/bdu83rosuwtku9y/iconUndo_active.svg
https://www.dropbox.com/s/zvvz2t8h3bbxv50/iconUndo.svg

pathway-undo

**Comment by stenington** [comment]
Thanks @zzzndr!

**Pull request merged**